movie/src/components/ActionContainer/ActionContainer.jsx

```diff
@@ -0,0 +1,103 @@
+import "./ActionContainer.css";
+import SortCard from "../SortCard/SortCard";
+import FilterCard from "../FilterCard/FilterCard";
+import { useState } from "react";
+
+const ActionContainer = ({ movies, onSearch }) => {
+  /**
+   * setSelectedGenres: store genres which we click at BoxDisplayGenres.jsx.
+   * setSortByOption: store option which we choose at SortPopup.jsx.
+   */
+  const [selectedGenres, setSelectedGenres] = useState([]);
+  const [sortByOption, setSortByOption] = useState("");
+
+  
+  const handleSortByOption = (option) => {
+    setSortByOption(option);
+  };
+
+  const handleClickFilter = (selected) => {
+    setSelectedGenres(selected);
+  };
+
+  // compareGenres is used to get movie after compare genres which I clicked and genres of movies.
+  const compareGenres = () => {
+    // However I don't select (selectGenres), it still return value (movies.)
+    if (selectedGenres.length === 0) return movies;
+    // When I clicked, I can get value to put in array (genresClicked)
+    const genresClicked = movies.filter((item) => {
+      return selectedGenres.find((arr) => item.genres.includes(arr));
+    });
+    return genresClicked;
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let movieAfterFilterOrSort = compareGenres();
+    switch (sortByOption) {
```

Comment:
can we store this switch function in a file. `./src/util/sort.js`, something like this